static-sites.yml

```diff
@@ -24,7 +24,7 @@ all:
   cinescuela.es:
     name: Cinescuela
     version: 2016-10-20
-    description: Lleva el cine a tu aula de clase.
+    description: Lleva el cine a tu biblioteca móvil.
```

Comment:
What if people install this package in another context than the current Colombia deployment?